code/modules/mob/living/carbon/human/human_update_icons.dm

```diff
@@ -126,7 +126,14 @@ There are several things that need to be remembered:
 		var/mutable_appearance/uniform_overlay
 
 		if(dna && dna.species.sexes)
-			var/G = (gender == FEMALE) ? "f" : "m"
+			var/G
+			switch(gender)
+				if(MALE)
+					G = "m"
+				if(FEMALE)
+					G = "f"
+				else
+					G = (body_type == MALE ? "m" : "f")

```

Comment:
can we rename these magic strings to just MALE and FEMALE too? looks like G only matters for 130.

Reply:
done

Reply:
done